src/services/corsproxy/corsproxy.service.ts

```diff
@@ -17,7 +17,8 @@ export class WizdomCorsProxyService implements IWizdomCorsProxyService {
             this.corsProxyState = window["WizdomCorsProxyState"];
         }
 
-        if (message.command === "WizdomCorsProxyFailed") {            
+        if (message.command === "WizdomCorsProxyFailed" && !window["WizdomCorsProxyState"].session) {   
+            console.error("Corsproxy failed to initialize");         
```

Comment:
This means that if for some reason it fails on recreating the frame after the token is expired it will still think it's working because it has a session ID from the first frame

Reply:
Also there's no need to do this because this event basically just refreshes the state from the shared state on window, making it not refresh might cause things to get out of sync